packages/block-library/src/image/block.json

```diff
@@ -4,7 +4,14 @@
 	"name": "core/image",
 	"title": "Image",
 	"category": "media",
-	"usesContext": [ "allowResize", "imageCrop", "fixedHeight", "postId", "postType", "queryId" ],
+	"usesContext": [
+		"allowResize",
+		"imageCrop",
+		"fixedHeight",
+		"postId",
+		"postType",
+		"queryId"
+	],
```

Comment:
This change looks unintentional and unrelated to the PR - could you revert it before we ship the PR?

Reply:
Hi @tyxla, I would unable to do so, because when I sync the current branch with trunk and rebase it, and did commit, it was done automatically when run linting using precommit hooks.